packages/components/src/slot-fill/bubbles-virtually/slot-fill-provider.js

```diff
@@ -93,6 +93,7 @@ function useSlotRegistry() {
 			slots,
 			fills,
 			registerSlot,
+			updateSlot,
```

Comment:
This was also missing from the React memo deps, although it wasn't making any difference.